src/handlers/lib/utility.js

```diff
@@ -433,9 +433,10 @@ const produceGeneralMessage = async (functionality, action, message, state, key)
     functionalityMapped = Enum.topicMap[functionality][action].functionality
     actionMapped = Enum.topicMap[functionality][action].action
   }
-  return await Kafka.Producer.produceMessage(updateMessageProtocolMetadata(message, functionality, action, state),
+  let result = await Kafka.Producer.produceMessage(updateMessageProtocolMetadata(message, functionality, action, state),
     createGeneralTopicConf(functionalityMapped, actionMapped, key),
```

Comment:
Why is the result assigned to a variable and then returned? Could it not be done in one line with return await?